book.toml

```diff
@@ -1,2 +1,3 @@
 [output.html]
-no-section-label = true
\ No newline at end of file
+no-section-label = true
+google_analytics_id = "UA-144588868-2"
```

Comment:
Thinking of keeping a separate analytics for RFCs. Lets use 'UA-144588868-3' Here.

Reply:
ack